docs/sdk.md

```diff
@@ -70,12 +70,12 @@ world over gRPC, via the implicit [gRPC pseudo-Node](concepts.md#pseudo-nodes)
 that has a channel to the Application's initial Node.
 
 An Oak Node that interacts with gRPC typically implements the
-[`oak::grpc::OakNode`](https://project-oak.github.io/oak/sdk/oak/grpc/trait.OakNode.html)
+[`oak::grpc::OakGrpcServerNode`](https://project-oak.github.io/oak/sdk/oak/grpc/trait.OakGrpcServerNode.html)
```

Comment:
I don't think this rename is necessary: the module name already indicates that `oak::grpc::OakNode` is gRPC-specific.

Reply:
I wanted to emphasize that it is a server though. How about `ServerNode` (omitting the `OakGrpc` prefix)?

Reply:
SGTM

Reply:
How about `derive(OakExports)` -> `derive(GrpcServerEntrypoint)`?